jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/util/CaseMilestoneStatusEnum.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.cm.client.util;
+
+import java.util.Arrays;
+
+public enum CaseMilestoneStatusEnum {
+
+    AVAILABLE("Available", "Available"),
+
+    COMPLETED("Completed", "Completed"),
+
+    TERMINATED("Terminated", "Terminated");
+
+    private String label;
```

Comment:
status and label seem to be exact the same, I believe there is no need to duplicate in this case.